web/console/src/api/queue.ts

```diff
@@ -46,4 +46,16 @@ export class QueueClient {
             this.sendAction(action, squadId);
         };
     };
+
+    /** opens and initialize connection */
+    public openConnection() {
+        this.ws.onopen = () => {
+            this.ws.send('hello');
```

Comment:
add TODO if this code is test

Reply:
it is not test code, i need it to start websocket connection